README.md

```diff
@@ -1 +1,97 @@
 # jeux
+---
+Game night.
+We all love it, we all do it, we all have a special "gamer tag" that defines who we are in that space.
+Ever wanted to try out a game to see if it's worth the price or the time? I bet you have. We have the solution for you!
+Jeux is a handy site that lets you and your friends share games with one another while allow you to keep track of who has what. No more wondering where your Friday night specialty has gone! Jeux has you covered!
+
+## Table of Contents
+---
+- [What's inside (Technologies)](#What's-inside-Technologies)
```

Comment:
to correctly navigate these will need anchor tags <a name="of-the-thing-to-navigate-to"> </a>  no spaces no errant characters  as in ' / etc